onset_maproom/layout.py

```diff
@@ -164,97 +183,120 @@ def controls_layout():
-                      "days"
-                  )
+            Block("Latitude / longitude inputs",
+                dbc.Row(
+                    [
+                        dbc.Col(
+                            html.Div([dbc.Input(
+                                id = "latInput", 
+                                placeholder="Type latitude...", 
+                                type="number",
+                                min=3,max=15
+                            ),
+                            dbc.FormText("lat range: 3-15N")]),
+                        ),
+                        dbc.Col(
+                            html.Div([dbc.Input(
+                                id = "lngInput",
+                                placeholder="Type longitude...",
+                                type="number",
+                                min=33,max=48
+                            ),
+                            dbc.FormText("lng range: 33-48E")]), 
+                        ),
+                        dbc.Button(id="submitLatLng", n_clicks=0, children='Submit'),
+                    ],
+                ),
             ),
-
-            Block("Wet Day Definition",
-                  Sentence(
-                      "Rainfall amount greater than",
-                      Number("wetThreshold", 1, min=0, max=99999),
-                      "mm",
-                  )
+            #Block(
+            #    "Date",
+            #    dbc.Select(
+            #        id="date_input",
+            #        value="onset",
+            #        bs_size="sm",
+            #        options=[
+            #            {"label": "Onset", "value": "onset"},
+            #            {"label": "Cessation", "value": "cessation"},
+            #        ],
+            #    ),
+            #),
+           # Block(
+           #     "Yearly Statistics",
+           #     dbc.Select(
+           #         id="yearly_stats_input",
+           #         value="mean",
+           #         bs_size="sm",
+           #         options=[
+           #             {"label": "Mean", "value": "mean"},
+           #             {"label": "Standard deviation", "value": "stddev"},
+           #             {"label": "Probability of exceedance", "value": "pe"},
+           #         ],
+           #     ),
+           #     dbc.Collapse(
+           #         Sentence(
+           #             Number("probExcThresh1", 30, min=0, max=999),
+           #             "days since Early Start, as",
+           #             Units("poeunits"),
+           #         ),
+           #         id="probability-collapse",
+           #     ),
+           # ),
+            Block(
+                "Onset Date Search Period",
+                Sentence(
+                    "From Early Start date of",
+                    Date("search_start_", 1, "Jun"),
+                    "and within the next",
+                    Number("searchDays", 90, min=0, max=9999), "days",
```

Comment:
this one text box seems to be stuck to a smaller size that prevent proper reading of the number